addons/mail/static/src/model/model_manager.js

```diff
@@ -1156,12 +1100,13 @@ export class ModelManager {
                 continue;
             }
             hasChanged = true;
-            // flag all dependent fields for compute
-            this._registerComputeOfDependents(record, field);
+            // Mark listeners to notify.
+            for (const listener of this._listenersUsingLocalIdField.get(record.localId).get(field)) {
+                this._markListenerToNotify(listener);
+            }
```

Comment:
If you need a comment for a 2/3 LOCs, the LOCs are not clear enough.
Either the code is clear enough and just remove the comment, or define a well-named function & params (and remove the comment too).

Reply:
This comment was indeed pretty useless, I'll just remove it.